### ISSUE TITLE
Using filter results in incorrect word count

### ISSUE THREAD

**Issue by tamias** [w8tcha/CKEditor-WordCount-Plugin#234]
WordCount plugin version 1.17.9.

When the word count plugin is configured with a filter, whitespace between tags is stripped out, resulting in an incorrect word count.

**Behavior With No Filter**

HTML source:
```
<p>One</p>

<p>Two</p>

<p>Three</p>
```

Normalized to:
```
<p>One</p>  <p>Two</p>  <p>Three</p>
```

Stripped to:
```
One Two Three
```

Resulting in a word count of 3.

**Behavior With A Filter**

Configure a filter:
```
  config.wordcount = {
    filter: new CKEDITOR.htmlParser.filter(),
  };
```

HTML source:
```
<p>One</p>

<p>Two</p>

<p>Three</p>
```

Normalized to:
```
<p>One</p>  <p>Two</p>  <p>Three</p>
```

Filtered to:
```
<p>One</p><p>Two</p><p>Three</p>
```

Stripped to:
```
OneTwoThree
```

Resulting in a word count of 1.

**Comment by tamias** [comment]
It looks like the change made in af2f2f025 to fix #232 might fix this issue.  However, it hasn't been released yet...

**Comment by w8tcha** [comment]
I uploaded version 1,17.10. I cannot reproduce the issue with that version.
